src/components/editor/InputLiteral.jsx

```diff
@@ -23,6 +23,15 @@ export class InputLiteral extends Component {
     }
   }
 
+  componentDidMount = () => {
+    if (this.props.propertyTemplate.repeatable === "false" &&
+        this.props.formData !== undefined &&
+        this.props.formData.items.length > 0) {
+      this.setState( { disabled: true })
```

Comment:
is there a way to test this?  It's sorta what you're actually fixing ... seems worth having a test to show if we break it in future ...

Reply:
Done in latest commit